code/modules/admin/topic.dm

```diff
@@ -1582,6 +1582,9 @@
 		var/atom/movable/AM = locate(href_list["admingetmovable"])
 		if(QDELETED(AM))
 			return
+		if(!check_rights(R_FUN))
+			if(AM.z == 1)
```

Comment:
WOoP WOOP SHITCODE ALERT use `is_centcom_level(AM.z)` ree

also consider allowing teleporting from centcom to centcom.